### ISSUE TITLE
CT only components that were added should be removed

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/energy-skate-park-basics#446]
```
energy-skate-park-basics : phet-io-fuzz : require.js : run
Uncaught Error: Assertion failed: Only components that were added should be removed
Error: Assertion failed: Only components that were added should be removed
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/assert/js/assert.js:22:13)
    at PhetioEngine.removeInstance (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/phet-io/js/phetioEngine.js?bust=1544139621943:560:17)
    at Object.removeInstance (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/phet-io/js/phetioEngine.js?bust=1544139621943:592:46)
    at Tandem.removeInstance (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/tandem/js/Tandem.js?bust=1544139621943:145:34)
    at Track.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/tandem/js/PhetioObject.js?bust=1544139621943:354:21)
    at Track.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/energy-skate-park-basics/js/energy-skate-park-basics/model/Track.js?bust=1544139621943:923:38)
    at Track.PhetioObject.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/tandem/js/PhetioObject.js?bust=1544139621943:128:22)
    at Array.<anonymous> (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/energy-skate-park-basics/js/energy-skate-park-basics/model/EnergySkateParkBasicsModel.js?bust=1544139621943:189:13)
    at ObservableArray._fireItemRemoved (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/axon/js/ObservableArray.js?bust=1544139621943:162:18)
    at ObservableArray.remove (https://bayes.colorado.edu/continuous-testing/snapshot-1544137087182/axon/js/ObservableArray.js?bust=1544139621943:202:14)
id: Bayes Chrome
Approximately 12/6/2018, 3:58:07 PM
```

**Comment by samreid** [comment]
@jessegreenberg should I work on this before the merge?

**Comment by jessegreenberg** [comment]
Thanks @samreid, feel free to wait on this if you wish, I am going to work on https://github.com/phetsims/energy-skate-park/issues/6 tomorrow.

**Comment by samreid** [comment]
On hold until https://github.com/phetsims/energy-skate-park/issues/6 is complete.

**Comment by jessegreenberg** [comment]
@samreid https://github.com/phetsims/energy-skate-park/issues/6 has been addressed.

**Comment by samreid** [comment]
addressed in https://github.com/phetsims/energy-skate-park/issues/43, closing.
